crypto/tmhash/hash.go

```diff
@@ -21,6 +21,20 @@ func Sum(bz []byte) []byte {
 	return h[:]
 }
 
+// SumMany takes at least 1 byteslice along with a variadic
+// number of other byteslices and produces the SHA256 sum from
+// hashing them as if they were 1 joined slice.
+// Its purpose is to avoid programmers from thinking of inefficient patterns
+// like manually joining byteslices before hashing them altogether.
```

Comment:
```suggestion
```

It's sufficient to add this to the issue's / PR description.

Reply:
Gotcha, I've removed it. Thank you @melekes!